### PR TITLE
Support non-standard cross-browser manifest fields #122

### PR DESCRIPTION
**Pull request by joeframbach**

### CODE REVIEW

**Comment by joeframbach** [comment]
At https://github.com/extension-js/extension.js/blob/main/programs/cli/spec/build.spec.ts#L29-L32, I wanted to also validate the templates are creating the correct per-browser manifests, but I did not do this.

**Comment by joeframbach** [comment]
Fixed https://github.com/extension-js/extension.js/blob/main/packages/manifest-compat-plugin/handleRuntimeErrors/handleFirefoxRunningServiceWorkerError.ts#L10. This line would fail when the `background` key doesn't exist.

File: programs/create/templates/preact-typescript/template/manifest.json
Comment (cezaraugusto) [inline programs/create/templates/preact-typescript/template/manifest.json:1]: thanks for updating the templates as well!

**Review by cezaraugusto — APPROVED** [review]
++ this is very cool, thanks a lot for the contribution @joeframbach! 

closes #122

**Comment by amal** [comment]
@cezaraugusto, can you release this feature, please!?  Can't wait to start using it.

**Pull request merged**